src/vignette/util/thumbnail.clj

```diff
@@ -54,7 +54,9 @@
         args (reduce conj base-command thumb-options)
         sh-out (run-thumbnailer args)]
     (cond
-      (zero? (:exit sh-out)) (io/file temp-file)
+      (or (zero? (:exit sh-out))
+          (and (= 1 (:exit sh-out))
+               (file-exists? temp-file))) (io/file temp-file)
```

Comment:
Nice to have: a comment about why we are doing this. Not knowing the context, it looks odd with the `1` magic number.